src/manager.js

```diff
@@ -43,7 +43,14 @@ export default class Manager {
                     return fs.readFileAsync(configFilePath);
                 })
                 .then((buffer) => {
-                    this.config = JSON.parse(buffer.toString());
+                    const config = JSON.parse(buffer.toString());
+
+                    // Normalize all paths in the config object
+                    Object.keys(config.projects).forEach(repo => {
+                      config.projects[repo].path = path.resolve(config.projects[repo].path);
```

Comment:
Now about path.resovle, I definitely see why an absolute path would be cumbersome. However i wanted multi-git to be usable from anywhere. For your solution to work, you would have to run multi-git in the same folder as .mg-config.

So, instead of running path.resolve in the cwd, i'ld rather join the path of the parent folder of the current config and each repo path. Does that make sense ?
This should'nt be much harder since we have the path in the promise chain 😸 

That way, if you have the following tree:
```
/home
    /users
        /someone
            .mg-config.json
            /workspace
                /project1
                /project2
                /sub-workspace
                    /project3
                    /project4
/etc
    /wtf-folder
        /project5
```
the following .mg-config.json:
```
{
    "projects": {
            "project1": {
            "name": "project1",
            "path": "./workspace/project1"
        },
            "project2": {
            "name": "project2",
            "path": "./workspace/project2"
        },
            "project3": {
            "name": "project3",
            "path": "./workspace/sub-workspace/project3"
        },
            "project4": {
            "name": "project4",
            "path": "./workspace/sub-workspace/project4"
        },
            "project5": {
            "name": "project5",
            "path": "/etc/wtf-folder/project5"
        }
    }
}
```
running `multi-git st` would still yield the same result from `/home/users/someone` or `/etc/wtf-folder`